engine/launcher/general.cpp

```diff
@@ -38,17 +41,22 @@ int RANDOMIZED = 0;
 
 #ifdef _G_STRING_PARSE
 
-char *next_parm(char *string) {
-	char *next;
-	if (string[0] == '\0') { return 0; }
-	next = string;
-	while (next[0] != ' ' && next[0] != '\0') {
-		next++;
-	}
-	if (next[0] == '\0') { return next; }
-	next[0] = '\0';
-	next++;
-	return next;
+char *next_parm(char *string)
+{
+    char *next;
+    if (string[0] == '\0') {
```

Comment:
While personally I like having the braces on separate lines, it's more important that we stick with one method for all blocks - either all like Line 47 or all like lines 44-45.

Reply:
OK. I think I was keeping the braces on the same line for everything except for function definitions. But I can adjust that.